service-front/module/Application/view/application/pages/cp/confirm_dob.twig

```diff
@@ -21,7 +21,7 @@
             </div>
             <div class="govuk-form-group">
```

Comment:
This should be a fieldset/legend like [in the documentation](https://design-system.service.gov.uk/patterns/dates/), not a div/h2